ktor-client/ktor-client-plugins/ktor-client-platform/common/src/Platforming.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.plugins.platform
+
+import io.ktor.client.plugins.api.ClientPlugin
+import io.ktor.client.plugins.api.createClientPlugin
+import io.ktor.client.request.host
+import io.ktor.utils.io.errors.UnknownServiceException
+
+public val Platforming: ClientPlugin<Any> = createClientPlugin("Platforming", ::Any) {
```

Comment:
How can I install this plugin for CIO by default?

Reply:
let's make this in the separate PR, it looks like it require service loader or some similar mechanism

Reply:
We need

- https://youtrack.jetbrains.com/issue/KT-25892
- https://github.com/whyoleg/sweet-spi